internal_review/economics/PUMA/ACS_PUMS_economics_0812.csv

```diff
@@ -0,0 +1,56 @@
.9,1.8,1.0,769,250.0,19.8,5.0,1.6,1343,354.0,16.0,8.7,2.2,2183,374,10.4,14.2,2.2,5798,691,7.2,37.6,3.6,1428,485.0,20.6,9.3,3.1,558,188,20.5,3.6,1.2,767,244.0,19.4,5.0,1.5,6333,717,6.9,100,,0,,,,,13,29.0,137.9,0.2,0.5,69,90.0,78.9,1.1,1.4,42,58.0,83.3,0.7,0.9,564,256,27.6,8.9,3.9,824,277.0,20.5,13.0,4.1,176,113.0,39.1,2.8,1.8,483,211,26.6,7.6,3.2,421,181.0,26.1,6.6,2.8,2588,470,11.0,40.9,5.8,359,174.0,29.4,5.7,2.7,436,194.0,27.0,6.9,3.0,358,193.0,32.7,5.7,3.0,20249,1230,3.7,100,,14,26.0,113.9,0.1,0.1,695,242.0,21.2,3.4,1.2,954,341,21.8,4.7,1.7,466,239.0,31.1,2.3,1.2,2221,537,14.7,11.0,2.6,1176,406,21.0,5.8,2.0,595,278.0,28.4,2.9,1.4,1979,489,15.0,9.8,2.3,1977,448,13.8,9.8,2.1,6017,735,7.4,29.7,3.1,2283,542,14.4,11.3,2.6,1333,447,20.4,6.6,2.2,539,291,32.8,2.7,1.4,2716,439,9.8,100,,0,,,,,55,70.0,77.6,2.0,2.6,113,103.0,55.3,4.2,3.7,49,61.0,76.1,1.8,2.2,225,154.0,41.5,8.3,5.5,71,82.0,70.6,2.6,3.0,101,86.0,51.9,3.7,3.1,225,116.0,31.4,8.3,4.1,159,114.0,43.8,5.9,4.1,1221,352,17.5,45.0,10.7,113,109,58.5,4.2,3.9,343,159,28.2,12.6,5.5,41,65.0,96.1,1.5,2.4,45689,1985,2.6,100,,68022,4509,4.0,,,23957,3282,8.3,,,37299,4226,6.9,,,40101,11934,18.1,,,31425,4264,8.2,,,15401,1104,4.4,100,,81088,6654,5.0,,,27791,8732.0,19.1,,,50225,6837.0,8.3,,,59472,29593.0,30.2,,,41867,6843.0,9.9,,,6333,717,6.9,100,,57112,8842,9.4,,,32411,6175.0,11.6,,,40101,14429.0,21.9,,,89735.0,81798.0,55.4,,,41342,13882.0,20.4,,,20249,1230,3.7,100,,51926,7003,8.2,,,20357,2924.0,8.7,,,32117,4530,8.6,,,35149.0,9819.0,17.0,,,25991,5064.0,11.8,,,2716,439,9.8,100,,76272,13622,10.9,,,17097,7211,25.6,,,54324,65083.0,72.8,,,20949.0,47307.0,137.3,,,32516.0,6210.0,11.6,,,45689,1985,2.6,100,,32678.0,,,,,28960,10481,22.0,,,39557,14363,22.1,,,43531,16945,23.7,,,22089,9671,26.6,,,43866,8088,11.2,,,73833,12424,10.2,,,54376,10567,11.8,,,49501,6201,7.6,,,46612,3714,4.8,,,24455,7334,18.2,,,29012,5592,11.7,,,76335,14648,11.7,,,15401,1104,4.4,100,,97639.0,,,,,29827,27871.0,56.8,,,66875.0,31244.0,28.4,,,64720.0,68795.0,64.6,,,45533.0,9772.0,13.0,,,74440,24132.0,19.7,,,81751.0,24275.0,18.1,,,73015.0,17068.0,14.2,,,62583,18934.0,18.4,,,69955,7425.0,6.5,,,26445.0,11897.0,27.3,,,48795,22906.0,28.5,,,101162.0,22043.0,13.2,,,6333,717,6.9,100,,,,,,,75321.0,,,,,68884.0,63997.0,56.5,,,51106.0,,,,,16178,16320.0,61.3,,,57568.0,13690.0,14.5,,,91299.0,26380.0,17.6,,,43044,11104.0,15.7,,,42252.0,22388.0,32.2,,,43364,4840,6.8,,,36577.0,8956.0,14.9,,,29756.0,5755.0,11.8,,,58630.0,17931.0,18.6,,,20249,1230,3.7,100,,30685.0,,,,,29158.0,13961.0,29.1,,,29400,8171,16.9,,,37034.0,12439.0,20.4,,,19635,9438.0,29.2,,,32663,7870,14.6,,,46505.0,13587.0,17.8,,,50524,12820,15.4,,,39358,13618,21.0,,,30544,6050,12.0,,,20250,4388,13.2,,,14984,11831.0,48.0,,,62975,19779.0,19.1,,,2716,439,9.8,100,,,,,,,19868.0,19846.0,60.7,,,31775.0,122955.0,235.2,,,58911.0,53644.0,55.4,,,45268.0,90909.0,122.1,,,11000.0,,,,,91536.0,48657.0,32.3,,,43990.0,89289.0,123.4,,,70114.0,65402.0,56.7,,,67518,25593,23.0,,,33764,40233.0,72.4,,,41147,43207.0,63.8,,,125536.0,,,,
```

Comment:
I think the educational attainment indicators are off from the 2000 census pums economic column names. These are the original col and the renamed cols used in the mapper: 
```
    "lths": "edu_lths",
    "hsgrd": "edu_hsgrd",
    "sclga": "edu_sclga",
    "bchd": "edu_bchd",
```